apps/crawler-feeds/src/app.py

```diff
@@ -41,7 +42,8 @@ def main():
     otef_lev_feed = OtefLevFeed()
     minster_of_health_mental_clinic = MOH_MentalHealthClinicsFeed()
     maccabi_feed = MaccabiTherapyClinicsFeed()
-
+    clalit_feed = ClalitFeed()
```

Comment:
I think we should pass args in the feed and pull many services of clalit

Reply:
Good thinking.
Will be done in new assignments in the future.